### PR TITLE
Refactored to use environment variables for configuration.

### PR DESCRIPTION
**Pull request by CarlHembrough**
See readme updates for details.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,21 @@
...
 Microservice to load datapoints into the Data Discovery database, driven by Kafka messages.
 
+### Configuration
+
+The port that the application runs on needs to be passed as an option to the JVM `-Dhttp.port=20097`
+When running locally the default port is set in the .sbtops file
+
+| Environment variable | Default | Description
+| -------------------- | ------- | -----------
+| APPLICATION_SECRET   | changeme                                         | Required by Play framework
```
Comment (NeilMadden) [inline README.md:12]: Does anything actually use this?

File: README.md [hunk 2]
```diff
@@ -12,13 +27,30 @@ You will need to create a postgres database and user to run the tests out of the
...
+```
+
+
+### Contributing
+
+See [CONTRIBUTING](CONTRIBUTING.md) for details.
+
+### License
+
+Copyright ©‎ 2016, Office for National Statistics (https://www.ons.gov.uk)
```
Comment (NeilMadden) [inline README.md]: 2016-2017 now? ;-)

**Review by NeilMadden — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (CarlHembrough) [reply README.md:12]: No its something baked into the Play framework. I could not see anything obvious to remove the need for it.

File: README.md (hunk 2 above)
Comment (CarlHembrough) [reply README.md]: doh

**Pull request merged**